web/styles/popovers.css

```diff
@@ -1284,15 +1326,27 @@ ul {
     }
 }
 
+#user_card_popover {
+    --popover-menu-max-width: 250px;
```

Comment:
I have also set a max-width of `250px` to prevent the custom profile fields
from extending too far, this also might need some dicussion.

Reply:
That sounds prudent.